www/index.html

```diff
@@ -217,17 +218,16 @@ <h3>Expert settings</h3>
                                 <div class="panel panel-danger" id="contentInjectionModeDiv">
                                     <div class="panel-heading">Content injection mode</div>
                                     <div class="panel-body">
-                                        Don't touch unless you know what you're doing!
                                         <div class="radio">
                                             <label>
                                                 <input type="radio" name="contentInjectionMode" value="jquery" id="jQueryModeRadio" checked>
-                                                <strong>JQuery</strong> (slow and memory hungry, but safer)
+                                                <strong>JQuery</strong> (stable and safer)
                                             </label>
                                         </div>
                                         <div class="radio">
                                             <label>
                                                 <input type="radio" name="contentInjectionMode" value="serviceworker" id="serviceworkerModeRadio">
-                                                <strong>ServiceWorker</strong> (faster but unstable, and not supported by all platforms)
+                                                <strong>ServiceWorker</strong> (still a bit experimental, and not supported by all platforms)
```

Comment:
> still a bit experimental, and not supported by all platforms 

English is fine, but maybe could be more concise: "experimental, not supported by all platforms".

Reply:
To me, the SW mode is now usable in most cases (when available). It's not really "experimental" any more, even if we know a few edge cases that cause problems.
So I'd keep the "still a bit", unless you have a better way to say that?

Reply:
@mossroy: OK, so maybe "Somewhat experimental …."? I think "Still a bit" implies that the user knows how it used to be, which is not necessarily the case. But I'm fine with "Still a bit" if you prefer it.

Reply:
OK, updated with "somewhat experimental" and pushed